README.md

```diff
@@ -1150,7 +1150,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Shiladitya Bose](https://github.com/shiladityab24)
   - [Shivam Choudhary](https://github.com/Shivam-opensrc2002)
   - [Shivam Katare](https://github.com/Shivam-Katare)
-  - [Shivaraj M](https://github.com/Shivaraj21)
+  - [Shivam Sharma](https://github.com/shivam-sharma7)
```

Comment:
It seems you removed Shivaraj and added your name instead.
I think you need to add your name and keep the other one.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient